app/helpers/gtl_helper.rb

```diff
@@ -53,6 +53,7 @@ def render_gtl(options)
       :hideSelect        => options[:selected_records].kind_of?(Array),
       :showUrl           => gtl_show_url(options),
       :pages             => options[:pages],
+      :noFlashDiv        => options[:no_flash_div], ##
```

Comment:
@MelsHyrule options[:no_flash_div] is true for catalog page?

Reply:
Yep! Tested out on `Services > Catalogs > Catalog` and got no double flash